src/index.js

```diff
@@ -40,7 +40,10 @@ export function isJSONfile(url) {
   return /\.json5?$/.test(url);
 }
 
-export function transformJSONtoSass(json) {
+export function transformJSONtoSass(json, fileName) {
+  if (json.constructor === Array) {
+    json = { [fileName]: json };
+  }
```

Comment:
This is great! Just one thing... The `filename` argument is only relevant in an edge case. I think the JSON parsing might as well happen outside this function. That way we can keep `transformJSONtoSass` the same: it simply expects a JSON object.

Something like:

```js
try {
  const fileContents = require(filename);
  const extensionlessFilename = basename(fileName, extname(fileName));
  const json = Array.isArray(fileContents) ? { [extensionlessFilename]: fileContents } : fileContents;

  return {
    contents: transformJSONtoSass(json),
  }
```

Reply:
Perfect. I agree. I added the changes.
Thank you.